openedx/features/caliper_tracking/tests/expected/edx.problem.hint.demandhint_displayed.json

```diff
@@ -0,0 +1,47 @@
+{
+    "@context": "http://purl.imsglobal.org/ctx/caliper/v1p1",
+    "action": "Viewed",
+    "actor": {
+        "id": "http://localhost:18000/u/honor",
+        "name": "honor",
+        "type": "Person"
+    },
+    "eventTime": "2018-10-18T13:41:20.592Z",
+    "extensions": {
+        "extra_fields": {
+            "accept_language": "en;q=1.0, en;q=1.0",
+            "agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.67 Safari/537.36",
+            "asides": {},
+            "course_id": "course-v1:edX+DemoX+Demo_Course",
+            "course_user_tags": {},
+            "event_source": "server",
+            "event_type": "edx.problem.hint.demandhint_displayed",
+            "host": "3c31c9770429",
+            "ip": "172.18.0.1",
+            "module": {
+                "display_name": "Text Input with Hints and Feedback",
+                "usage_key": "block-v1:TestUni+ITP47+2018_T3+type@problem+block@9cc0a1c1ee014c02ae6b543998877f1a"
+            },
+            "org_id": "edX",
+            "page": "x_module",
+            "path": "/courses/course-v1:edX+DemoX+Demo_Course/xblock/block-v1:TestUni+ITP47+2018_T3+type@problem+block@9cc0a1c1ee014c02ae6b543998877f1a/handler/xmodule_handler/hint_button",
+            "user_id": 6
+        }
+    },
+    "id": "urn:uuid:13bc41f6-8406-4006-b929-c9f1ac156ada",
+    "object": {
+        "id": "http://localhost:18000/courses/course-v1:TestUni+ITP47+2018_T3/courseware/49442fdda84645d0950f613ca3052292/becccc9761f544aabd448a9167ef51a2/1?activate_block_id=block-v1%3ATestUni%2BITP47%2B2018_T3%2Btype%40vertical%2Bblock%400c93cd8fba454a9d810d5901265ef6c5",
+        "type": "Document",
```

Comment:
Is type document appropriate in this situation?

Reply:
changed to  'Frame'